jetstream/config.py

```diff
@@ -432,7 +499,7 @@ def resolve(
 
             stats_params = copy.deepcopy(params)
             pre_treatments = []
-            for pt in stats_params.pop("pre_treatments", []):
+            for pt in stats_params.pop("pre_treatments", []):  # type: ignore
```

Comment:
@scholtzan I'm not sure why suddenly `mypy` started complaining about this line. It doesn't seem to be related to my change as far as I can tell. Without the ignore I get the following error now:
```
venv/bin/mypy jetstream     
jetstream/config.py:507: error: Item "None" of "Union[str, Dict[str, Any], None, List[<nothing>]]" has no attribute "__iter__" (not iterable)
Found 1 error in 1 file (checked 43 source files)
```

My first thought was that maybe it was related to dependency update, but it seems like the checks passed fine before. What do you think?

Reply:
It still succeeds for me when I do the check locally. I'm guessing it only fails when you do the check locally (since the CI check here is fine)? Did you install the most recent mypy version locally?

Reply:
Oh I see the previous commit that failed.

Reply:
https://app.circleci.com/pipelines/github/mozilla/jetstream/1999/workflows/acf03629-6b12-429f-bddc-8430f7164d76/jobs/4295/parallel-runs/0/steps/0-106

Reply:
There error here is probably correct (albeit confusing). There is a problem with multiple variable having the name `params` (see comment above)